File: src/test/java/application/history/endpoint/history/api/CreateHistoryTest.java
package application.history.endpoint.history.api;

import application.history.IntegrationTest;
import application.history.WithFaker;
import application.history.history.application.services.CreateHistoryService;
import lombok.SneakyThrows;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;


public class CreateHistoryTest extends IntegrationTest implements WithFaker {

    @Autowired
    private CreateHistoryService service;

    @SneakyThrows
    @Test
    public void getAllHistories() {
    }
}

File: src/history/test/kotlin/com/example/history/application/history/endpoint/history/api/CreateHistoryTest.kt
package com.example.history.application.history.endpoint.history.api

import application.history.IntegrationTest
import org.junit.jupiter.api.Test

class CreateHistoryTest : IntegrationTest(), WithFaker {
    @Autowired
    private val service: CreateHistoryService? = null

    @get:Test
    @get:SneakyThrows
    val allHistories: Unit
        get() {}
}
